### ISSUE TITLE
Closing brackets should not be allowed to go multiline

### ISSUE THREAD

**Issue by rafaellehmkuhl** [vuejs/eslint-plugin-vue#2483]
**Please describe what the rule should do:**
Closing brackets should not be allowed to go multiline.

**What category should the rule belong to?**
[x] Enforces code style (layout)

```vue
<div class="mb-10 slide-right-to-left">
  <v-btn fab small>
    <v-tooltip open-delay="600">Cancel</v-tooltip
    ><v-icon>mdi-close</v-icon></v-btn
  >
</div>
```

The previous situation, where both `v-tooltip` and `v-btn` have their `>` in a newline, is currently accepted.
The following should be enforced:

```vue
<div class="mb-10 slide-right-to-left">
  <v-btn fab small>
    <v-tooltip open-delay="600">Cancel</v-tooltip>
    <v-icon>mdi-close</v-icon>
  </v-btn>
</div>
```

Am I missing something or there's no rule to enforce this yet?
